diff -ur a/source/creator/CMakeLists.txt b/source/creator/CMakeLists.txt
--- a/source/creator/CMakeLists.txt	2019-08-03 03:05:15.868212669 +0200
+++ b/source/creator/CMakeLists.txt	2019-08-03 03:05:38.677971585 +0200
@@ -351,12 +351,6 @@
 # Install Targets (Generic, All Platforms)
 
 
-# important to make a clean  install each time, else old scripts get loaded.
-install(
-  CODE
-  "file(REMOVE_RECURSE ${TARGETDIR_VER})"
-)
-
 if(WITH_PYTHON)
   # install(CODE "message(\"copying blender scripts...\")")
 
